lib/cretonne/src/isa/intel/mod.rs

```diff
@@ -111,4 +117,149 @@ impl TargetIsa for Isa {
     fn reloc_names(&self) -> &'static [&'static str] {
         &binemit::RELOC_NAMES
     }
+
+    fn prologue_epilogue(&self, func: &mut ir::Function) -> result::CtonResult {
```

Comment:
I think the whole body of this method should be moved into `intel/abi.rs` to keep the top-level module clean. See `legalize_signature()`.